sdk/servicebus/azure-servicebus/tests/test_queues.py

```diff
@@ -1253,4 +1253,27 @@ def test_queue_cancel_scheduled_messages(self, servicebus_namespace, servicebus_
                     print(str(m))
                     m.complete()
                 raise
-            
\ No newline at end of file
+
+    @pytest.mark.liveTest
+    @pytest.mark.live_test_only
+    @CachedResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
+    @ServiceBusQueuePreparer(name_prefix='servicebustest', dead_lettering_on_message_expiration=True)
+    def test_github_uamqp_issue_135(self, servicebus_namespace, servicebus_namespace_key_name,
```

Comment:
Echoing comment from above with suggestion to potentially save you time/effort.  As before ignore if you prefer different verbiage.

```suggestion
    def test_queue_reconnect_send_after_long_wait(self, servicebus_namespace, servicebus_namespace_key_name,
```